zrml/swaps/rpc/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [dependencies]
-jsonrpsee = { version = "0.15.1", features = ["server", "macros"] }
-parity-scale-codec = { default-features = false, version = "3.0.0" }
```

Comment:
I mean why did we use `default-features = false` before? We are not in a no_std env. So we didn't really use `std` functionality, that's why we were doing this, right? 

https://github.com/paritytech/parity-scale-codec/blob/e960eb9be27059ab0458cef68b80f9ed0db6e757/Cargo.toml#L40

Reply:
I don't know why we did use it before, probably because the author assumed that we are in `no_std` env. Since `default` was disabled before, `std` was not loaded.
AFAIK the `std` environment is not necessary (unless you run into duplicate impl errors), but the implementations in `std` are more efficient than the `no_std` implementations.